tools/compiler/Cargo.toml

```diff
@@ -29,3 +29,6 @@ clap = { workspace = true }
 proc-macro2 = "1.0.11"
 spin_on = { workspace = true }
 itertools = { workspace = true }
+
+[target.'cfg(not(target_env = "msvc"))'.dependencies]
+jemallocator = "0.3.2"
```

Comment:
also, this is the candidate for a workspace dependency.

Reply:
I tried that, but couldn't get it to work - i.e. I moved this to the top level `Cargo.toml` and then wrote `.workspace.dependencies` instead of `.dependencies` but then cargo bails:

```
cargo build -p slint-viewer --release
error: failed to parse manifest at `/mnt/data1/milian/kdab/slint/Cargo.toml`

Caused by:
  this virtual manifest specifies a `target` section, which is not allowed
```

Reply:
You should keep the `[target.'cfg(not(target_env = "msvc"))'.dependencies]` in this file. But not have it in the top level Cargo.toml

Reply:
ah so only the version is moved to the top level `Cargo.toml`?

Reply:
yes